diff --git a/node_modules/@11ty/eleventy/src/Eleventy.js b/node_modules/@11ty/eleventy/src/Eleventy.js
index 719fa43..1aafe58 100644
--- a/node_modules/@11ty/eleventy/src/Eleventy.js
+++ b/node_modules/@11ty/eleventy/src/Eleventy.js
@@ -563,6 +563,7 @@ Arguments:
     benchmark.before();
     await this._initWatchDependencies();
     benchmark.after();
+    this.config.events.emit("watchInit");
   }
 
   /**
@@ -735,10 +736,13 @@ Arguments:
       EleventyErrorHandler.logger = this.logger;
     }
 
+    this.config.events.emit("beforeBuild");
+
     try {
       let promise = this.writer.write();
 
       ret = await promise;
+      this.config.events.emit("afterBuild");
     } catch (e) {
       EleventyErrorHandler.initialMessage(
         "Problem writing Eleventy templates",
