diff --git a/node_modules/vite/dist/node/chunks/dep-7817f5b4.js b/node_modules/vite/dist/node/chunks/dep-7817f5b4.js
index c5bee63..337ad79 100644
--- a/node_modules/vite/dist/node/chunks/dep-7817f5b4.js
+++ b/node_modules/vite/dist/node/chunks/dep-7817f5b4.js
@@ -21558,9 +21558,13 @@ function buildHtmlPlugin(config) {
                 }
                 // for each encountered asset url, rewrite original html so that it
                 // references the post-build location.
+                const namedOutput = Object.keys(config?.build?.rollupOptions?.input || {});
                 for (const attr of assetUrls) {
                     const value = attr.value;
-                    try {
+                    // Replace leading slash in content
+                    if (value.content !== '' && !namedOutput.includes(value.content) && !namedOutput.includes(value.content.replace(/^\//, ''))) {
+                      try {
+                      
                         const url = attr.name === 'srcset'
                             ? await processSrcSet(value.content, ({ url }) => urlToBuiltUrl(url, id, config, this))
                             : await urlToBuiltUrl(value.content, id, config, this);
@@ -21570,9 +21574,12 @@ function buildHtmlPlugin(config) {
                         // #1885 preload may be pointing to urls that do not exist
                         // locally on disk
                         if (e.code !== 'ENOENT') {
+                          console.log(attr);
                             throw e;
                         }
                     }
+                    }
+                    
                 }
                 processedHtml.set(id, s.toString());
                 // inject module preload polyfill only when configured and needed
