diff --git a/node_modules/vite-plugin-posthtml/index.js b/node_modules/vite-plugin-posthtml/index.js
index 12b5701..c14913b 100644
--- a/node_modules/vite-plugin-posthtml/index.js
+++ b/node_modules/vite-plugin-posthtml/index.js
@@ -29,7 +29,7 @@ const posthtmlPlugin = (opts = {}) => {
 
     performance.clearMarks();
   });
-  observer.observe({ entryTypes: ['measure'], type: 'measure' });
+  observer.observe({ entryTypes: ['measure'] });
 
   return {
     name: 'posthtml',
